acceptance/repository-mysql/src/__tests__/mysql.datasource.ts

```diff
@@ -13,7 +13,7 @@ export const MYSQL_CONFIG: DataSourceOptions = {
   port: process.env.MYSQL_PORT || 3306,
   database: process.env.MYSQL_DATABASE || 'repository-tests',
   username: process.env.MYSQL_USER || 'root',
-  password: process.env.MYSQL_PASSWORD || '',
+  password: process.env.MYSQL_PASSWORD || 'pass',
```

Comment:
This change seems unrelated to "move relation tests to repository-tests package". Can you please revert it here and open a new PR to discuss it in isolation?